src/index.ts

```diff
@@ -4,6 +4,7 @@ export { Checkbox } from './Checkbox';
 export { Divider } from './Divider';
 export { Icon } from './Icon';
 export { Radio } from './Radio';
+export { Tab } from './Tab';
```

Comment:
I think we need to add type-checking to the CI. There's a typo here:
```suggestion
export { Tabs } from './Tabs';
```